### PR TITLE
Added bd5a device and changed flag device name

### PR DESCRIPTION
**Pull request by MDecarabas**

### CODE REVIEW

File: src/instrument/devices/bd_5a.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+from ophyd import Device
+from ophyd import EpicsMotor
+from ophyd import FormattedComponent as FCpt
+
+logger = logging.getLogger(__name__)
+logger.info(__file__)
+
+
+class BD5A(Device):
```
Comment (prjemian) [inline src/instrument/devices/bd_5a.py:20]: Could this be simpler?

```py
class BD5A(Device):
    h = Component(EpicsMotor, "m9", labels={"motors"})
    v = Component(EpicsMotor, "m10", labels={"motors"})

bd5a = BD5A("8iddSoft:CR8-D1:US:", name="bd-5a")
```

Comment (prjemian) [reply src/instrument/devices/bd_5a.py:20]: I'm thinking that `BD5A` is not intended for any re-use.  Is that right?

File: src/instrument/devices/flag_4.py [hunk 2]
```diff
@@ -36,8 +36,8 @@ def __init__(
     flag_4 = FCpt(EpicsMotor, "{motor_prefix}:{_flag_4_motor}", labels={"motors"})
 
 
-fl4 = Flag4(
-    name="fl4",
+flag4 = Flag4(
```
Comment (prjemian) [inline src/instrument/devices/flag_4.py]: This creates a motor object `flag4.flag_4`.  To move it, the user might use `yield from bps.mvr(flag4.flag_4, -0.05)`.  Is there any advantage to encapsulating this motor inside a device?

File: src/instrument/devices/bd_5a.py (hunk 1 above)
Comment (MDecarabas) [reply src/instrument/devices/bd_5a.py:20]: Possibly, but I like the idea of the format being exactly the same as in all the other places that have a device with multiple motors

File: src/instrument/devices/flag_4.py (hunk 2 above)
Comment (MDecarabas) [reply src/instrument/devices/flag_4.py]: fixed

**Review by prjemian — APPROVED** [review]
The code works, but looks complicated.  I see the change in `flag4`.

@qzhang234 Any comments on this?

**Pull request merged**